### PR TITLE
Revise Amazon S3 data source

### PR DESCRIPTION
**Pull request by vagimeli**
### Description
REvises text for clarity and conciseness

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._

### Version
_List the OpenSearch version to which this PR applies, e.g. 2.14, 2.12--2.14, or all._

### Frontend features
_If you're submitting documentation for an OpenSearch Dashboards feature, add a video that shows how a user will interact with the UI step by step. A voiceover is optional._ 

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _dashboards/management/S3-data-source.md [hunk 1]
```diff
@@ -10,51 +10,41 @@ has_children: true
...
 
-## Connect your Amazon S3 data source 
+## Connect your data source 
 
-To connect your Amazon S3 data source, follow these steps:
+To connect your data source, follow these steps:
 
-1. From the OpenSearch Dashboards main menu, select **Management** > **Data sources**.
-2. On the **Data sources** page, select **New data source** > **S3**. An example UI is shown in the following image.
+1. From the OpenSearch Dashboards main menu, go to **Management** > **Data sources**.
```
Comment (vagimeli) [inline _dashboards/management/S3-data-source.md:27]:
```suggestion
1. From the OpenSearch Dashboards main menu, go to **Management** > **Dashboards Management** > **Data sources**.
```

File: _dashboards/management/S3-data-source.md [hunk 2]
```diff
@@ -10,51 +10,41 @@ has_children: true
...
-## Connect your Amazon S3 data source 
+## Connect your data source 
 
-To connect your Amazon S3 data source, follow these steps:
+To connect your data source, follow these steps:
 
-1. From the OpenSearch Dashboards main menu, select **Management** > **Data sources**.
-2. On the **Data sources** page, select **New data source** > **S3**. An example UI is shown in the following image.
+1. From the OpenSearch Dashboards main menu, go to **Management** > **Dashboards Management** > **Data sources**.
+2. On the **Data sources** page, select **New data source** > **S3**.
```
Comment (vagimeli) [inline _dashboards/management/S3-data-source.md:28]:
```suggestion
2. On the **Data sources** page, select **Create data source connection** > **Amazon S3**.
```

File: _dashboards/management/S3-data-source.md [hunk 3]
```diff
@@ -10,51 +10,41 @@ has_children: true
...
+## Connect your data source 
 
-To connect your Amazon S3 data source, follow these steps:
+To connect your data source, follow these steps:
 
-1. From the OpenSearch Dashboards main menu, select **Management** > **Data sources**.
-2. On the **Data sources** page, select **New data source** > **S3**. An example UI is shown in the following image.
+1. From the OpenSearch Dashboards main menu, go to **Management** > **Dashboards Management** > **Data sources**.
+2. On the **Data sources** page, select **Create data source connection** > **Amazon S3**.
+3. On the **Configure Amazon S3 data source** page, enter the necessary details under **Data source details**, **AWS Glue authentication details**, **AWS Glue index store details**, and **Query permissions**.
```
Comment (vagimeli) [inline _dashboards/management/S3-data-source.md:29]:
```suggestion
3. On the **Configure Amazon S3 data source** page, enter the data source and authentication details and permissions.
```

File: _dashboards/management/S3-data-source.md [hunk 4]
```diff
@@ -10,51 +10,41 @@ has_children: true
...
 
-To connect your Amazon S3 data source, follow these steps:
+To connect your data source, follow these steps:
 
-1. From the OpenSearch Dashboards main menu, select **Management** > **Data sources**.
-2. On the **Data sources** page, select **New data source** > **S3**. An example UI is shown in the following image.
+1. From the OpenSearch Dashboards main menu, go to **Management** > **Dashboards Management** > **Data sources**.
+2. On the **Data sources** page, select **Create data source connection** > **Amazon S3**.
+3. On the **Configure Amazon S3 data source** page, enter the data source and authentication details and permissions.
+4. Verify the configuration by selecting **Review Configuration**.
```
Comment (vagimeli) [inline _dashboards/management/S3-data-source.md:30]:
```suggestion
4. Select **Review Configuration** to verify the connection details.
```

File: _dashboards/management/S3-data-source.md [hunk 5]
```diff
@@ -10,51 +10,41 @@ has_children: true
...
+## Connect your data source 
 
-To connect your Amazon S3 data source, follow these steps:
+To connect your data source, follow these steps:
 
-1. From the OpenSearch Dashboards main menu, select **Management** > **Data sources**.
-2. On the **Data sources** page, select **New data source** > **S3**. An example UI is shown in the following image.
+1. From the OpenSearch Dashboards main menu, go to **Management** > **Dashboards Management** > **Data sources**.
+2. On the **Data sources** page, select **Create data source connection** > **Amazon S3**.
+3. On the **Configure Amazon S3 data source** page, enter the data source and authentication details and permissions.
```
Comment (Naarcha-AWS) [inline _dashboards/management/S3-data-source.md:29]:
```suggestion
3. On the **Configure Amazon S3 data source** page, enter the data source, authentication details, and permissions.
```

**Review by Naarcha-AWS — APPROVED** [review]

**Review by natebower — COMMENTED** [review]
@vagimeli Just a few minor changes. Thanks!

File: _dashboards/management/S3-data-source.md [hunk 6]
```diff
@@ -10,51 +10,41 @@ has_children: true
...
-Once you've connected your Amazon S3 data source, you can explore your data through the **Manage data sources** tab. The following steps guide you through using this functionality:
+To manage your data source, follow these steps:
 
 1. On the **Manage data sources** tab, choose a date source from the list. 
-2. On that data source's page, you can manage the data source, choose a use case, and manage access controls and configurations. An example UI is shown in the following image.
-
-    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/manage-data-source-UI.png" alt="Manage data sources UI" width="700"/>
-
-3. (Optional) Explore the Amazon S3 use cases, including querying your data and optimizing query performance. Go to **Next steps** to learn more about each use case.
+2. On the data source's page, you can manage the data source, choose a use case, and configure access controls.
```
Comment (vagimeli) [inline _dashboards/management/S3-data-source.md:38]:
```suggestion
2. On the page for the data source, you can manage the data source, choose a use case, and configure access controls. 
```

**Pull request merged**